Tests/AppTests/QueryPerformanceTests.swift

```diff
@@ -115,6 +115,7 @@ class QueryPerformanceTests: XCTestCase {
     }
 
     func test_12_Search_refresh() async throws {
+        throw XCTSkip("run only after schema change with new index has been deployed")
```

Comment:
I'll update the search figure once I've deployed this change from the branch. It needs to have the index in place. Without it, the query takes 30s locally on my machine vs 200ms with it in place.

Reply:
Actually, I don't need to deploy, I can just temporarily create the index manually for this test.